jplag.frontend-utils/src/main/java/de/jplag/TokenPrinter.java

```diff
@@ -24,19 +24,21 @@
  * @author Timur Saglam
  */
 public final class TokenPrinter {
+    public static final String NON_WHITESPACE = "\\S";
+    public static final int MIN_PADDING = 1;
     private static final Logger logger = LoggerFactory.getLogger(TokenPrinter.class);
     // Representation:
     private static final String BAR = "|";
     private static final String TAB = "\t";
     private static final String SPACE = " ";
-    public static final String NON_WHITESPACE = "\\S";
-    public static final int MIN_PADDING = 1;
```

Comment:
i'm not sure right now if moving the lines of code here makes sense. Was there a reason to move them?

Reply:
That must have been the Spotless plugin. You are right, that looks weird.

Reply:
I didn't set them public on purpose, so it's good that this came up.